### PR TITLE
Fixed #7 Added the mentors section

### LINKED ISSUE #7: Create a mentors section
We just added a new template
Create a mentors section in the current template with the same design.

The goal would be to create a mentor section, add a mentors.yml file, define a standard format, and make a mentor appear beautifully in the site using Jekyll.

_note: be sure to ask mentors before posting any of their details_

Ref: https://GitHub.com/fossasia/gci17.fossasia.org

### PR DESCRIPTION
**Pull request by STEdirisuriya**
I used jekyll and added mentors section as this (issue[https://github.com/fossasia/gci18.fossasia.org/issues/7]).
So I `jekyll build` it for first time. So It shows lot of changes in this PR
Thanks!

### CODE REVIEW

**Comment by STEdirisuriya** [comment]
https://stedirisuriya.github.io/gci18.fossasia.org/ Here is the preview link.

**Comment by STEdirisuriya** [comment]
@mariobehling I had to use jekyll. Because of that I build the site with jekyll to view it from the link.

**Comment by STEdirisuriya** [comment]
I did changes

**Comment by mariobehling** [comment]
A mentors section needs to contain - mentors. Simply adding placeholders does not make sense. We already have sample html in the repo.

**Comment by YashKumarVerma** [comment]
@mariobehling  sir,
maybe he wants to add a ***mentors section*** and not mentors to the page. 
@STEdirisuriya only one sample data is enough for reference.

**Comment by STEdirisuriya** [comment]
@mariobehling and @YashKumarVerma Thanks, I added six mentors from gci17 site Have a look now.

**Comment by abishekvashok** [comment]
@STEdirisuriya just add one mentor, who is a mentor this year as well. We don't need past years mentor names here.

**Comment by fragm3** [comment]
Yes, rest looks good to be.

**Comment by isuruAb** [comment]
@mariobehling  please review

**Comment by abishekvashok** [comment]
> @STEdirisuriya just add one mentor, who is a mentor this year as well. We don't need past years mentor names here.

**Comment by STEdirisuriya** [comment]
@mariobehling please have a look again!

**Comment by sarnava1** [comment]
LGTM @mariobehling.

**Comment by STEdirisuriya** [comment]
> > @STEdirisuriya just add one mentor, who is a mentor this year as well. We don't need past years mentor names here.

File: Gemfile [hunk 1]
```diff
@@ -4,4 +4,3 @@ gem 'fastimage'
 gem 'img_checker'
 #gem 'jekyll-sitemap'
 #gem 'jekyll-paginate'
-
```
Comment (liveHarshit) [inline Gemfile]: Extra. No need to change in this file.

**Comment by liveHarshit** [comment]
Open the website, mentors section is already there. You can change UI or upgrade it by creating a new issue and send another PR.

**Comment by STEdirisuriya** [comment]
@liveHarshit Resolved

**Comment by STEdirisuriya** [comment]
> Open the website, mentors section is already there. You can change UI or upgrade it by creating a new issue and send another PR.

I claimed the issue first and make the PR. Then how there is a another one? Another PR get merged first. That was the reason. Kindly give me a solution.

**Comment by tabesin** [comment]
@liveHarshit  On the deployed https://gci18.fossasia.org/ I don't see a mentor section, maybe you mean the gci17 site?

**Comment by STEdirisuriya** [comment]
> @liveHarshit On the deployed https://gci18.fossasia.org/ I don't see a mentor section, maybe you mean the gci17 site?

He is telling about an another PR. But there is a problem. I claimed the issue first and make the PR.

**Comment by tabesin** [comment]
Guys, this is the comment section about this PR which is about the mentor section for the GCI18 site. The review of Mario has been resolved as far as I could see as only 10 files are changed.

I also don't see an existing mentor section at the deployed site and there seem to be no merge conflicts.

Other PR and other repositories are irrelevant here.

**Comment by liveHarshit** [comment]
This branch has conflicts that must be resolved - 
Conflicting files
index.html

**Comment by STEdirisuriya** [comment]
> This branch has conflicts that must be resolved -
> Conflicting files
> index.html

Resolved. Please review now @liveHarshit

**Comment by liveHarshit** [comment]
Drop your 2 commits. Then rebase it after that again commit it with update mentor section message then push it. It is still showing conflicts.

**Comment by STEdirisuriya** [comment]
@liveHarshit OK. resolved again

**Comment by liveHarshit** [comment]
I told you to change commit message to `Update mentor section (#7)`

**Comment by STEdirisuriya** [comment]
> I told you to change commit message to `Update mentor section (#7)`

OK. Did

**Pull request merged**